### ISSUE TITLE
Errors when trying to add nextcloud integration

### ISSUE THREAD

**Issue by CristianKerr** [alainm23/planify#1157]
**Describe the bug**
I tried to connect the nextcloud that I use (https://cloud.woelkli.com/login) but I am getting errors when trying different urls,
I basically copy following (edited) url:
https://cloud.woelkli.com/remote.php/dav/calendars/my-username/my-calendar-name
And I tried to also reduce the url by one fragment at a time and input that (all the way to https://cloud.woelkli.com/) and I always got one of those two errors on screenshot below.

**To Reproduce**
1. Create account on https://cloud.woelkli.com/login 
2. Create a calendar and add some tasks in
3. In tasks, next to the calendar name, click on ... and choose Copy private link
4. paste the link into planify integrations and add your username and password
5. See error

**Expected behavior**
The tasks are loaded.

**Screenshots**
![Screenshot from 2024-02-27 09-16-06](https://github.com/alainm23/planify/assets/79404036/9310eb04-d1b1-4a40-888b-6aff32d6ced9)
![Screenshot from 2024-02-27 09-15-03](https://github.com/alainm23/planify/assets/79404036/dd4133e5-0bd4-49d0-9964-706d83773df4)

**Desktop (please complete the following information):**
 - OS or DE: Gnome (on Fedora 39)
 - Version  45.4

**Additional context**
-

**Comment by underdoeg** [comment]
I think you should only set the Server URL to ```https://cloud.woelkli.com```

**Comment by CristianKerr** [comment]
> I think you should only set the Server URL to `https://cloud.woelkli.com`

I tried that as well, it threw same error shown on the first screenshot

**Comment by underdoeg** [comment]
No, from what I can see your first screenshot points to ```https://cloud.woelkli.com/remote.php``` ?

**Comment by CristianKerr** [comment]
> No, from what I can see your first screenshot points to `https://cloud.woelkli.com/remote.php` ?

I know, I am saying that I tried without remote.php as well and it threw the same error as is on screenshot one. As I write in the original post, I tried the whole path in every variation (by always removing the last fragment), so I tried:
https://cloud.woelkli.com/remote.php/dav/calendars/my-username/my-calendar-name
https://cloud.woelkli.com/remote.php/dav/calendars/my-username
https://cloud.woelkli.com/remote.php/dav/calendars
https://cloud.woelkli.com/remote.php/dav
https://cloud.woelkli.com/remote.php
https://cloud.woelkli.com

**Comment by underdoeg** [comment]
Ah yes, I see it now, sorry. I just had similar errors to yours until I finally tried just the root URL.

**Comment by CristianKerr** [comment]
No problem :)

**Comment by alainm23** [comment]
It is only necessary to enter the base URL.

**Comment by CristianKerr** [comment]
> It is only necessary to enter the base URL.

That doesn't work either.

**Comment by alainm23** [comment]
I've just created an account, I'm debugging it.

**Comment by CristianKerr** [comment]
thanks a lot! 💛

**Comment by alainm23** [comment]
@CristianKerr  This server has some special details, when using Postman I can log in without problem. However, when using Planify it returns a code 500 error.

**Comment by CristianKerr** [comment]
I see, maybe some header is needed? Dunno, if its too much trouble let me know and let it be, I was considering selfhosting nextcloud so maybe this is the incentive needed for me to finally do it 😊

**Comment by byquanton** [comment]
Tested with a current build of planify this works now without any issues.
